Tests/ImageTintingTests.swift

```diff
@@ -84,5 +86,23 @@ class ImageTintingTests: XCTestCase {
 
         BONAssertEqualImages(untintedResult!, tintAttemptResult!)
     }
+    
+    func testAccessibilityIOS() {
+        #if os(iOS)
```

Comment:
Same as above: what about tvOS?
Also: you could just do a single `testAccessibility` function, which the contents wrapped in `#if`, instead of two separate functions. I don't strongly care either way, I guess, but it might let you share more lines of code between the tests.